src/main/java/tictactoe/Game.java

```diff
@@ -36,7 +38,7 @@ public String outcome() {
         if (board.winningLineExists(currentPlayersMark(), opponentsMark())) {
             return winner();
         } else {
-            return "tie";
+            return GameOutcome.TIE.type;
```

Comment:
Has this brought about any advantages over the previous use of a `String` type?

Reply:
I missed this. Thanks for spotting it. I'll change it

Reply:
I'd say that it's easier to search for in my code, compared to searching for the string `"tie"`, `"h"` or `"c"`